redhat/opm-simulators.spec

```diff
@@ -114,14 +116,15 @@ Requires:       ptscotch-mpich
 %description mpich-bin
 This package contains the applications for opm-simulators
 
+%global debug_package %{nil}
+
 %prep
 %setup -q -n %{name}-release-%{version}-%{tag}
 
 %build
-scl enable devtoolset-6 bash
```

Comment:
or was this what you were referring to markus? this line is void and a leftover after prior attempts. each line in the spec executes in a separate shell so it has no effect. this is why i have to pass the compilers manually below.